prov/sm2/src/sm2.h

```diff
@@ -296,6 +297,11 @@ struct sm2_ep {
 	void *dsa_context;
 };
 
+static inline struct sm2_region *sm2_smr_region(struct sm2_ep *ep, int id)
+{
+	return sm2_mmap_ep_region(ep->mmap_regions, id);
+}
+
```

Comment:
An inline function that calls one thing that is almost identical?

Reply:
removed